### DISCUSSION TITLE
Error in Proxmox 8.0 and LXC

### DISCUSSION THREAD

**Discussion by X-Anonymous-Y** [tteck/Proxmox#1590] [Help Needed] [answered]
Error in Proxmox 8.0!

No matter what script I use to create a container! does anyone of you have the same problem:

Example: `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/debian.sh)"`

Error in Bold!

Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 100
Using Hostname: debian
Using Disk Size: 2GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Debian LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 **- Updating LXC Template List...update failed - see /var/log/pveam.log for details**
 ✓ Updated LXC Template List
 ✓ LXC Container 10030 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.178.1
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

SomeSuperUser@Proxmox-VE:~# format -rf C:\

**Comment by tteck** [comment, marked as answer]
>  - Updating LXC Template List...update failed - see /var/log/pveam.log for details

This is because https://releases.turnkeylinux.org/pve/ is down

**Reply by X-Anonymous-Y** [reply]
That's right :D ... I could have figured it out myself! Thanks for the answer! I have tested it. and confirm my stupidity

**Reply by X-Anonymous-Y** [reply]
`View the logs cat /var/log/pveam.log`

I did it and pinged the site too
